lib/world.rb

```diff
@@ -10,6 +10,10 @@ def as_json
     @_wjson ||= JSON.parse(File.read(file), symbolize_names: true)
   end
 
+  def countries
+    as_json.map { |slug, _| country(slug) }
```

Comment:
Would probably be a little clearer to do that as `as_json.keys.map { |slug| country_slug }`